### PR TITLE
Add support for Swift 5.6 branch

### PR DESCRIPTION
**Pull request by shahmishal**

### CODE REVIEW

**Comment by shahmishal** [comment]
@swift-ci test with toolchain

**Review by clackary — APPROVED** [review]

File: projects.json [hunk 1]
```diff
@@ -2204,7 +2204,7 @@
           {
             "issue": "https://bugs.swift.org/browse/SR-15109",
             "compatibility": ["5.0"],
-            "branch": ["main", "release/5.5"],
+            "branch": ["main", "release/5.5", , "release/5.6"],
```
Comment (shahmishal) [inline projects.json:2207]:
```suggestion
            "branch": ["main", "release/5.5", "release/5.6"],
```
Fix typo

**Comment by shahmishal** [comment]
@swift-ci test with toolchain

**Comment by shahmishal** [comment]
```
00:01:36.293 + ./swift-source-compat-suite/utils/format_projects_list.py swift-source-compat-suite/projects.json

```
Passed

**Pull request merged**